### PR TITLE
enca: clarify examples and fix tokens

### PR DESCRIPTION
**Pull request by mebeim**
- Clarify and simplify examples' descriptions.
- Change the token `{{files(s)}}` to `{{file1 file2 ...}}` for consistency with other pages.

### CODE REVIEW

File: pages/common/enca.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 > More information: <https://github.com/nijel/enca>.
 
-- Detect file(s) encoding according to your system's locale:
+- Detect file(s) encoding according to the system's locale:
 
-`enca {{file(s)}}`
+`enca {{file1 file2 ...}}`
 
-- Detect file(s) encoding; -L option tells enca the current language; language is in the POSIX/C locale format, e.g. zh_CN, en_US etc:
+- Detect file(s) encoding specifying a language the POSIX/C locale format (e.g. zh_CN, en_US):
```
Comment (andrik) [inline pages/common/enca.md]:
``` suggestion
Detect file(s) encoding specifying a language in the POSIX/C locale format (e.g. zh_CN, en_US):
```

Comment (sbrl) [reply pages/common/enca.md]: Looks  like the preceding dash got lost there, @andrik

Comment (mebeim) [reply pages/common/enca.md]: Fixed, thanks.

**Comment by mebeim** [comment]
To anyone who merges: please keep the full commit message of the first commit when squashing.

**Comment by mebeim** [comment]
Merging since it's approved and has been sitting here for a while.

**Pull request merged**